### DISCUSSION TITLE
Can't search the sentence "Consuming task queue - timeout"

### DISCUSSION THREAD

**Discussion by outloudvi** [community/community#23190] [New to GitHub]
<p>GitHub just don’t let me search “Consuming task queue - timeout” in any repository, saying " Code Search could not process your request, please try different search criteria". I tried in several more repositories and it gave the same result.</p>
<p>For example: <a href="https://github.com/syzoj/syzoj-ng/search?q=Consuming+task+queue+-+timeout" rel="noopener nofollow ugc">https://github.com/syzoj/syzoj-ng/search?q=Consuming+task+queue+-+timeout</a></p>
<p>(<a href="https://github.com/syzoj/syzoj-ng/blob/5e78251c08c63d8f25340aad4f51c87c4c0aac56/src/judge/judge-queue.service.ts#L101" rel="noopener nofollow ugc">This sentence did appear in this repository</a>.)</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/5/7/5757464aa01b8e977971b043339e283ea81ee461.png" data-download-href="/uploads/short-url/csEwxvxKUrpgvxBBSxZVKrzwfuh.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/19144373/181075536-a97553fa-0c1f-4998-9b55-98da62a4cb2a.png" alt="image" data-base62-sha1="csEwxvxKUrpgvxBBSxZVKrzwfuh" width="690" height="205" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/5/7/5757464aa01b8e977971b043339e283ea81ee461_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">714×213 15.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Is it “expected”, like this disagrees with GitHub’s search syntax?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>I think it’s the <code>-</code> causing the problem. In search syntax, that typically means you want to exclude results that contain the word following the <code>-</code>.</p>
<p>If you put the search term in quotes, you should find it works.</p>
<pre><code> https://github.com/syzoj/syzoj-ng/search?q=%22Consuming+task+queue+-+timeout%22&amp;type=code</code></pre>

**Comment by outloudvi** [comment]
<p>Got it, thanks for your kind reply!</p>
